components/admin_console/team_channel_settings/channel/details/channel_members/channel_members.tsx

```diff
@@ -174,6 +175,10 @@ export default class ChannelMembers extends React.PureComponent<Props, State> {
             if (channelRoles.length > 0) {
                 filters = {...filters, channel_roles: channelRoles};
             }
+            [...systemRoles, ...channelRoles].forEach((role) => {
+                trackEvent('admin_channel_config_page', `${role}_filter_applied_to_members_block`);
```

Comment:
@thefactremains do we want a different event for each role? or should I change this to a generic
`filters_applied_to_members_block` no matter what filters are applied?

Reply:
Discussed on mattermost decided to go with a different event for each filter for this and the other one